### ISSUE TITLE
Vignette explaining model

### ISSUE THREAD

**Issue by athowes** [epinowcast/epidist#64]
## Goal

This issue is for creation of a vignette going into greater detail about the model `latent_truncation_censoring_adjusted_delay`.

## Context

The `epidist` package focuses on a particular statistical model. How the model works is quite complex, and likely to be challenging to understand. However, better understanding of the model is likely to lead to better use of the package. Furthermore, there is scope for the model structure to be altered in specific ways within the remit of `epidst` by using `brms` formula interface. Users looking to make use of this functionality will likely need some kind of tutorial, and will particularly benefit from a good understanding of the model structure and implementation.

## Required features

- An `.Rmd` explaining the `latent_truncation_censoring_adjusted_delay` statistical model
- To include:
  - Its generative form
  - An explanation of how it accounts for censoring and right truncation
  - How it is implemented in the package
  - What the extra arguments of the function do, if you'd like to change them
  - Which parts of the arguments we would imagine you might want to change and the potential use-cases
  - The limitations of the model (ways it's not ideal)

## Related documents

- A summary of the maths in Park et al. https://athowes.github.io/cfa-notes/delay-dist-math
- https://github.com/epinowcast/epidist/pull/52 the Ebola vignette scoped to include spatial structure -- will be an example of playing with the model structure

**Comment by seabbs** [comment]
Yes good idea. I see this as part of scoping documentation (i.e do now/soon) vs help for a user (i.e do later/never).

> Which parts of the arguments we would imagine you might want to change and the potential use-cases
> The limitations of the model (ways it's not ideal)

These parts of the exercise seem particularly useful from a design perspective.

> A summary of the maths in Park et al

There is also a description in the Ward et al. paper that Park et a. cites but I find the former easier to follow/more correct

**Comment by athowes** [comment]
At the moment I think working this is a priority for me as I'm finding I'm blocked on things like "implement different delay distribution" and "implement stratified (space/sex) example" by my limited understanding of the model. I am also finding this to be important scoping documentation as Sam suggests, including for things like post-processing. More broadly I think it's quite important for me to understand where we are with `brms`, how much we can rely on it, how fragile it is, ...

**Comment by seabbs** [comment]
Yes makes sense

**Comment by athowes** [comment]
Closed branch https://github.com/epinowcast/epidist/pull/330 and tentatively assigned to @parksw3 for 1-2 weeks.

**Comment by parksw3** [comment]
Yes, I'll look at it today. Sorry for the delay. Had to prepare a presentation for today and didn't get a chance to get to this.

**Comment by parksw3** [comment]
I've made some progress. Wasn't sure if I should make a pull request yet or not so commenting here for now. Let's discuss tomorrow and will do more work. I also realized that we don't need latent variables for the secondary events and not sure how none of us noticed this earlier????

**Comment by seabbs** [comment]
Nice. Could throw up a draft PR for @athowes to take a gander at. 

> I also realized that we don't need latent variables for the secondary events and not sure how none of us noticed this earlier????

LOL. I feel like we noticed the distribution didn't matter and then took no further action...

**Comment by seabbs** [comment]
@SamuelBrand1 fancy acting as a notation reviewer for this as @parksw3 did on your write up?
